src/components/timeline/timeline.cy.tsx

```diff
@@ -0,0 +1,9 @@
+import { mount } from "@cypress/react18";
+import Timeline from "../timeline/timeline";
+
+describe("App.cy.tsx - test for Timeline", () => {
+  it("playground", () => {
+    mount(<Timeline />);
+    cy.get("[data-cy-id='timeline']").click().click();
```

Comment:
We actually get double "click" action here, but what next? what we testing here? :)

Reply:
We also need more tests here :)